### PR TITLE
CI: Invert path-ignore for tools folders

### PR DESCRIPTION
**Pull request by carlopi**
Idea is that it should be easier to track dependencies than keep up with other folders.

I have for sure forgot dependencies, will amend the PR accordingly.

### CODE REVIEW

File: .github/workflows/LinuxRelease.yml [hunk 1]
```diff
@@ -12,11 +12,8 @@ on:
   pull_request:
     paths-ignore:
       - '**.md'
-      - 'tools/jdbc/**'
-      - 'tools/nodejs/**'
-      - 'tools/juliapkg/**'
-      - 'tools/pythonpkg/**'
-      - 'tools/rpkg/**'
+      - 'tools/**'
+      - '!tools/pythonpkg/**'
```
Comment (Tmonster) [inline .github/workflows/LinuxRelease.yml:16]: why do we not ignore python here?

**Comment by Tmonster** [comment]
Just the one question, looks good to me if CI passes

File: .github/workflows/LinuxRelease.yml (hunk 1 above)
Comment (carlopi) [reply .github/workflows/LinuxRelease.yml:16]: You are right, unneded. Thanks for the catch

**Comment by Mytherin** [comment]
Thanks!

**Pull request merged**